x-pack/plugins/canvas/public/lib/aeroelastic/select.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ActionId, NodeFunction, NodeResult } from './types';
+
+export const select = (fun: NodeFunction): NodeFunction => (...fns) => {
+  let { prevId, cache } = { prevId: NaN as ActionId, cache: null as NodeResult };
```

Comment:
Isn't this just a simple default assignment? Wouldn't this be the same?

```js
let prevId: ActionId = NaN;
let cache: NodeResult = null;
```

Reply:
Yes, identical, they were kept together in one line as the two captured variables, I'm fine with switching to separate lines too